### PR TITLE
Defined schemas for JobExecutionSourceChunk entity

### PR DESCRIPTION
**Pull request by RuslanLavrov**

### CODE REVIEW

File: schemas/mod-source-record-manager/jobExecutionSourceChunk.json [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+  },
+  "required": [
+    "id",
+    "jobExecutionId",
+    "last",
+    "state",
+    "chunkSize",
+    "processedAmount",
+    "createdDate",
+    "completedDate"
```
Comment (OleksiiKuzminov) [inline schemas/mod-source-record-manager/jobExecutionSourceChunk.json]: not required

**Comment by OleksiiKuzminov** [comment]
plz add sample json to the examples folder

File: schemas/mod-source-record-manager/jobExecutionSourceChunk.json [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+      "type": "string",
+      "format": "date-time"
+    },
+    "error": {
+      "description": "Contains an error message and trace stack if the chunk processing has failed, makes sense only if State == Error",
+      "type": "string"
+    }
+  },
+  "required": [
+    "id",
```
Comment (KaterynaSenchenko) [inline schemas/mod-source-record-manager/jobExecutionSourceChunk.json]: Shouldn't we generate id upon receiving the chunk? not sure if this is required

Comment (Igor-Gorchakov) [reply schemas/mod-source-record-manager/jobExecutionSourceChunk.json]: Id is not required to receive, but it's required to store. Better to remove it from "required" section for now.

**Pull request merged**